rest-api/offline/public_metrics_export.py

```diff
@@ -0,0 +1,145 @@
+from sqlalchemy import text
+from dao import database_factory
+from dao.database_utils import replace_years_old
+from participant_enums import EnrollmentStatus, OrderStatus, PhysicalMeasurementsStatus
+from participant_enums import Race, QuestionnaireStatus
+
+# TODO(calbach): consider whether we need to filter down to consented
+# individuals only.
+
+def _questionnaire_metric(name, col):
+  """Returns a metrics SQL aggregation tuple for the given key/column."""
+  return (
+      name,
+      """
+      SELECT {}, SUM(1)
+      FROM participant_summary
+      GROUP BY 1;
+      """.format(col),
+      lambda v: QuestionnaireStatus.lookup_by_number(v).name
+  )
+
+# Metrics definitions. 3-tuples of:
+# - (str) aggregation key name
+# - (str) SQL statement to select value, count for a metric (in that order)
+# - (func(str): str) optional function which takes the value from the above SQL
+#   output and converts it for presentation
+_SQL_AGGREGATIONS = [
+  ('enrollmentStatus',
+   """
+   SELECT enrollment_status, SUM(1)
```

Comment:
I'm wondering if you should add a helper method that's like:

def ps_aggregation(field):
  return 'SELECT %s, COUNT(*) FROM participant_summary GROUP BY 1' % field

Then I think most of these would just be:

ps_aggregation('enrollment_status')

etc.